JPS_Ontology/ontology/ontodevice/OntoDevice.csv

```diff
@@ -36,7 +36,80 @@ FlowSensor,Class,https://saref.etsi.org/core/Sensor,IS-A,,,,A device that is use
 FeedbackSensor,Class,https://saref.etsi.org/core/Sensor,IS-A,,,,A device that is used to provide the control system with measurements of physical quantities necessary to close control loops.,https://www.theworldavatar.com/kg/ontodevice/,Feedback sensor
 ProximitySensor,Class,https://saref.etsi.org/core/Sensor,IS-A,,,,A device that takes a continuous distance reading and returns a corresponding analog voltage.,https://www.theworldavatar.com/kg/ontodevice/,Proximity sensor
 SashOpeningSensor,Class,https://saref.etsi.org/core/Sensor,IS-A,,,,A device that measures the sash opening of an object such as a fumehood.,https://www.theworldavatar.com/kg/ontodevice/,Sash opening sensor
+ConductivitySensor,Class,https://saref.etsi.org/core/Sensor,IS-A,,,,A device that measures the conductivity of a material.,https://www.theworldavatar.com/kg/ontodevice/,Conductivity sensor
+CurrentSensor,Class,https://saref.etsi.org/core/Sensor,IS-A,,,,A device that measures the current flowing through an electrical circuit.,https://www.theworldavatar.com/kg/ontodevice/,Current sensor
```

Comment:
You may want to clarify that it is _electrical_ currents that are meant here. There exist currents of many other things...